sdk/core/core-https/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Program",
+      "preLaunchTask": "Build Node",
+      "program": "${workspaceFolder}\\src\\index.ts",
```

Comment:
What would it do when launching index.ts? do we want `${file}` or something else to run a sample or test?

Reply:
I'm going to replace these with some better ones.